src/test/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinJvmTaskExecutorTest.kt

```diff
@@ -44,11 +44,11 @@ class KotlinJvmTaskExecutorTest {
 
     assertTrue(expandedCompileTask.hasInputs())
     assertNotNull(expandedCompileTask.inputs.javaSourcesList.find { path ->
-      path.endsWith("a_test_1/generated_sources/AnotherGenClass.java")
+      path.endsWith("generated_sources/AnotherGenClass.java")
```

Comment:
This always generates not `1` but a random number `a-test-115507972808607999568`. Removing the prefix here.